intro/matplotlib/examples/plot_grid.py

```diff
@@ -5,11 +5,12 @@
 An example displaying a grid on the axes and tweaking the layout.
 """
 
-import pylab as pl
+import numpy as np
```

Comment:
I tried to have automatics replacement (using emacs) and this is the result...
I think there are many scripts with the same problem :(
I'll try to correct that